src/constants/hub-enabled-country-list.ts

```diff
@@ -0,0 +1,85 @@
+export const HUB_ENABLED_COUNTRY_LIST = [
```

Comment:
do we need full hardcoded list as fallback?

Reply:
@habib-deriv can you please confirm.

Reply:
We kept the hardcoded list to make sure the website works works if Firebase API call is failed